src/main/java/com/elytradev/concrete/inventory/gui/GuiHelper.java

```diff
@@ -40,35 +40,44 @@
 import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
 import net.minecraft.util.ResourceLocation;
 
-public final class GuiDrawing {
-	private GuiDrawing() {}
+public final class GuiHelper {
+	private GuiHelper() {}
 	
-	public static void rect(ResourceLocation texture, int left, int top, int width, int height, int color) {
-		rect(texture, left, top, width, height, 0, 0, 1, 1, color);
+	public static void drawRectangle(ResourceLocation texture, int left, int top, int width, int height) {
+		drawRectangle(texture, left, top, width, height, 0xFFFFFFFF);
 	}
 	
-	public static void rect(ResourceLocation texture, int left, int top, int width, int height, float u1, float v1, float u2, float v2, int color) {
+	public static void drawRectangle(ResourceLocation texture, int left, int top, int width, int height, int color) {
+		drawRectangle(texture, left, top, width, height, 0, 0, 1, 1, color);
+	}
+	
+	public static void drawRectangle(ResourceLocation texture, int left, int top, int width, int height, float u1, float v1, float u2, float v2) {
+		drawRectangle(texture, left, top, width, height, u1, v1, u2, v2, 0xFFFFFFFF);
+	}
+	
+	public static void drawRectangle(ResourceLocation texture, int left, int top, int width, int height, float u1, float v1, float u2, float v2, int color) {
 		Minecraft.getMinecraft().getTextureManager().bindTexture(texture);
 		
-		//float scale = 0.00390625F;
+		//float scale = 00390625f;
```

Comment:
Yeah, sorry, I changed `0.0D` to `0.0` then decided to change it to plain `0`...